packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -692,9 +706,13 @@ const Breakouts = WebexPlugin.extend({
 
   /**
    * The pre-assignments need to be queried when "hasBreakoutPreAssignments" is true
+   * @param {Object} params
    * @returns {void}
    */
-  queryPreAssignments() {
+  queryPreAssignments(params) {
```

Comment:
issue: There should be new test cases for queryPreAssignments for this change

Reply:
Already modified test cases for the change